addons/mail/static/src/messaging/component/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -842,6 +842,120 @@ QUnit.test('open 2 different chat windows: enough screen width', async function
     );
 });
 
+QUnit.test('open 2 chat windows: check right shift operations are available', async function (assert) {
+    assert.expect(9);
+
+    this.data['mail.channel'].records = [
+        { id: 10, last_message: {id: 10} },
+        { id: 20, last_message: {id: 20} },
+    ];
+    this.data['mail.message'].records = [
+        {
+            body: 'bla',
+            channel_ids: [10],
+            id: 10,
+        },
+        {
+            body: 'blu',
+            channel_ids: [20],
+            id: 20,
+        },
+    ];
```

Comment:
Looks useless, no? Don't see any mention of messages in rest of test...

Reply:
The way `channel_fetch_preview` is mocked, it needs messages to be defined (and message component needs a body). In fact, we don't need channel definition, I'll remove them.

Reply:
I would adapt code of `_mockChannelFetchPreview` so that it works even when channels do not have any messages.